packages/amplify-graphql-api-construct-tests/src/__tests__/backends/amplify-table/simple-todo/updateIndex/app.ts

```diff
@@ -51,3 +51,9 @@ todoTable.setGlobalSecondaryIndexProvisionedThroughput('byName2', {
 todoTable.pointInTimeRecoveryEnabled = true;
 todoTable.sseSpecification = { sseEnabled: false };
 todoTable.streamSpecification = { streamViewType: StreamViewType.KEYS_ONLY };
+
+Tags.of(stack).add('created-by', 'amplify');
+Tags.of(stack).add('amplify:deployment-type', 'pipeline');
+Tags.of(stack).add('amplify:deployment-branch', 'main');
+Tags.of(stack).add('amplify:appId', '123456');
+Tags.of(stack).add('amplify:friendly-name', 'amplifyData');
```

Comment:
I believe you're mimicking tags that would be added by the backend? That's reasonable, but it does lead to some confusion in the test body itself, where it looks like deployments change from "sandbox" to "pipeline" for some reason. Instead, can we make the tag values clearly artificial (e.g., "original" and "modified") to make it clear you're testing that tags update as expected?

Reply:
Updated.